### DISCUSSION TITLE
== Error ==  system call  OS return value: -9

### DISCUSSION THREAD

**Discussion by andrewisaiah** [ablab/spades#1247] [Q&A]
Hi,

The spades run has return an Error return value -9. Everything seemed to be working fine prior to the error message. Below is the spades log file. 

 13:17:05.452   185G / 185G  INFO   K-mer Splitting          (kmer_data.cpp             :  97)   Processing /export2/home/andrew/Incubation/Peat/metagenome/concatanated/0_rawreads/Anoxic_R1_merged.fastq.gz
 13:18:59.599   185G / 201G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 11212655 reads
 13:20:56.632   185G / 201G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 22947503 reads
 13:22:52.818   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 34742220 reads
 13:24:56.206   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 46402407 reads
 13:27:02.094   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 57801051 reads
 13:29:16.681   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 69865822 reads
 13:31:31.729   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 81554074 reads
 13:33:55.101   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 93013501 reads
 13:36:14.755   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 104831003 reads
 13:38:33.095   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 116712418 reads
 13:40:49.269   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 128377006 reads
 13:42:53.722   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 139929359 reads
 13:44:52.721   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 151610404 reads
 14:08:31.960   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 274620377 reads
 14:16:15.919   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             :  97)   Processing /export2/home/andrew/Incubation/Peat/metagenome/concatanated/0_rawreads/Anoxic_R2_merged.fastq.gz
 15:00:03.677   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 107)   Processed 540243071 reads
 15:17:48.539   185G / 202G  INFO   K-mer Splitting          (kmer_data.cpp             : 112)   Total 627395088 reads processed
 15:17:48.566   163G / 202G  INFO    General                 (kmer_index_builder.hpp    : 249)   Starting k-mer counting.
 17:56:14.794   163G / 445G  INFO    General                 (kmer_index_builder.hpp    : 260)   K-mer counting done. There are 17103975968 kmers in total.
 17:56:37.214   133M / 445G  INFO   K-mer Index Building     (kmer_index_builder.hpp    : 395)   Building perfect hash indices
 18:19:11.108    12G / 445G  INFO   K-mer Index Building     (kmer_index_builder.hpp    : 431)   Index built. Total 17103975968 kmers, 12353602312 bytes occupied (5.77812 bits per kmer).
 18:19:11.109    12G / 445G  INFO   K-mer Counting           (kmer_data.cpp             : 354)   Arranging kmers in hash map order
 18:28:12.228   267G / 445G  INFO    General                 (main.cpp                  : 148)   Clustering Hamming graph.
 20:04:02.484   267G / 445G  INFO    General                 (main.cpp                  : 155)   Extracting clusters:
 20:04:02.484   267G / 445G  INFO    General                 (concurrent_dsu.cpp        :  18)   Connecting to root
 20:04:23.370   267G / 445G  INFO    General                 (concurrent_dsu.cpp        :  34)   Calculating counts

== Error ==  system call for: "['/export2/home/andrew/miniconda3/bin/spades-hammer', '/export2/home/andrew/Incubation/Peat/metagenome/concatanated/6_metaspades_assembly/Anoxic/corrected/configs/config.info']" finished abnormally, OS return value: -9
None

Param and Log file here:
[params.txt](https://github.com/ablab/spades/files/14354183/params.txt)
[spades.log](https://github.com/ablab/spades/files/14354184/spades.log)

Any recommendations would be highly appreciative

**Comment by asl** [comment]
Hello

Your SPAdes job was killed. Likely more information could be found in the system log.

**Reply by andrewisaiah** [reply]
Thanks I lot. I tried with a smaller file for assembly and it worked. The server I am using cannot handle large co-assemblies as there is not enough RAM space.

**Reply by asl** [reply]
Great you figured it out!
